### PR TITLE
x86_64 linux docker build

### PR DESCRIPTION
**Pull request by a1ecbr0wn**

### CODE REVIEW

**Comment by ogoffart** [comment]
Thanks for the PR!

I'm going to leave this to @tronical to review because he made the previous images.

I'm slightly concerned about bitrot since we do not test this. But maybe we could add one build for it in .github/workflows/embedded_build.yaml

File: docker/Dockerfile.x86_64-unknown-linux-gnu [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM rustembedded/cross:x86_64-unknown-linux-gnu
```
Comment (tronical) [inline docker/Dockerfile.x86_64-unknown-linux-gnu:1]: I think this should fix the CI error, if you substitute in your name/email :)

```suggestion
# Copyright © your name / your email
# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial

FROM rustembedded/cross:x86_64-unknown-linux-gnu
```

**Review by tronical — CHANGES_REQUESTED** [review]
Looks overall good to me - Thanks! I think it makes sense to have this. I think a little fix is needed for the license/copyright checker.

I agree with Olivier, it would be great if `embedded_build.yaml` could be extended to also use this - at least for CI purposes. But I can also do that.

**Comment by a1ecbr0wn** [comment]
Sorry, yep, missed that.

**Comment by a1ecbr0wn** [comment]
Made the changes, hope you are happy with it

**Review by tronical — APPROVED** [review]
Thanks

**Comment by ogoffart** [comment]
I tried to run ci with this image, and the build failed when trying to build the servo-fontconfig-sys crates:

https://github.com/slint-ui/slint/runs/6223849592?check_suite_focus=true

```
 thread 'main' panicked at '`"pkg-config" "--libs" "--cflags" "fontconfig" "fontconfig >= 2.11.1"` did not exit successfully: exit status: 1
  error: could not find system library 'fontconfig' required by the 'servo-fontconfig-sys' crate
```

looks like it is missing the fontconfig package.

**Comment by tronical** [comment]
I think that's just because the container is missing from `Cross.toml`. I'll look into it.

**Comment by a1ecbr0wn** [comment]
You are right that it is not in the `Cross.toml`, sorry I should have done this do you want me to submit another PR?

**Comment by tronical** [comment]
No worries, done in 7b0dab4ab1fd49cba37ab4827efe9dd93841d20d :)

**Pull request merged**